compiler/rustc_codegen_llvm/src/debuginfo/mod.rs

```diff
@@ -53,29 +53,31 @@ const DW_TAG_auto_variable: c_uint = 0x100;
 #[allow(non_upper_case_globals)]
 const DW_TAG_arg_variable: c_uint = 0x101;
 
+// `DW_OP_*` values taken from:
+// - `llvm/include/llvm/BinaryFormat/Dwarf.def`
+// - `llvm/include/llvm/BinaryFormat/Dwarf.h`
+#[allow(non_upper_case_globals)]
+const DW_OP_deref: u64 = 0x06;
+#[allow(non_upper_case_globals)]
+const DW_OP_plus_uconst: u64 = 0x23;
+#[allow(non_upper_case_globals)]
+const DW_OP_LLVM_fragment: u64 = 0x1000;
```

Comment:
can we glob these from a local module so we can apply the lint once on the module? it was tolerable when it was only 2, but